tests/functional/i/inconsistent/inconsistent_returns.py

```diff
@@ -1,5 +1,5 @@
 #pylint: disable=missing-docstring, no-else-return, no-else-break, invalid-name, unused-variable, superfluous-parens, try-except-raise
-#pylint: disable=disallowed-name
+#pylint: disable=disallowed-name,too-few-public-methods,no-member,undefined-variable,useless-else-on-loop
```

Comment:
```suggestion
#pylint: disable=disallowed-name,too-few-public-methods,useless-else-on-loop
```
This is a bit of a nit, but I'd prefer if the tests were just readable and functioning code where possible. I think these can be refactored to not give a `SyntaxError`